### DISCUSSION TITLE
Unable to fetch schema when running over Https and hosted over kubernetes

### DISCUSSION THREAD

**Discussion by kartikransi150** [ChilliCream/hotchocolate#4453] [Q&A]
Hi

Ive got my application running on Https which runs on kubernetes and uses kerberos authentication when connecting to database.

Now when Bana Cake pop or application tries to fetch schema it keeps trying and eventually fails.

From console logs i can see the GraphQL pipeline returns 404 for the requested route whichbis the application route. And i know it works because i can see the banana cake pop ide.

Any ideas as to what im doing wrong or need to configure ?

Note: ive already checked on google and stack overflow but had no luck.

**Comment by DuCNiKo** [comment]
Tricky (old) question...
First I'm using a biased version of kubernetes provided by the ops of my group (ie they have customized a lot of things) so my answer may be not be helpful... but:
- Check Kubernetes ingress (if you have one)
- Check kubernetes service

If you see the banana cake IDE, that means that you can perform a get on /graphql but can you perform a POST ? Which are the the queries performed by banana cake to get the SDL

Lastly, check in banana cake the graphql endpoint url. By default, it's /graphql


In your logs can you see the incoming post requests ?
No error on schéma building ?
